packages/docs/docs/getting-started/what-can-you-build-with-unirep.md

```diff
@@ -0,0 +1,29 @@
+---
+
+title: what can you build with unirep
+---
+
+# 🏗️ What Can You Build With UniRep
+
+UniRep is a generalized protocol that allow developers to build many different kinds of applications while keep user data private. Here are a few examples to illustrate the broad range of applications that could be built:
+
+Peer-to-Peer Anonymous Lending:
+ - A problem in current decentralized protocols is that most of them rely on the credit system being over-collateralized. With UniRep, the reputation value will be the credit-score of a prospective borrower that will change based on if they follow the credit rules of the prospective system.
+
+Reddit built with UniRep:
+- This example may be easiest to understand. The most important reputation value in Reddit is the karma user's earn. If we build Reddit with UniRep, the user's get to earn their karma anonymously while participtating in various different forums. You can then build in other "levels" into the platform when X karma gives you access to "Y", or grant a user the "mod" status or something of this nature.
+
+Anonymous GitHub:
+- Some developers may prefer to publish code anonymously. In A GitHub-type application built with UniRep, the reputation values could be the amount of commits a developer makes a repository, the amount of stars they have collectively on their profile, and/or if they have made successful PRs on a project. These reputation values can then be used to grant access to the anonymous developer to push to the main branch.
+
+Eth or Token Attesters:
+- It is possible to create an attester where the reputation value is Eth or some Token. This would allow the transfer of value anonymously. 
+
+Multi-Attester Applications:
```

Comment:
since the medical records ex is also a multi-attester example, it would be good to start with a description of the structure of a multi-attester app, then put the two examples under that subheading.